src/plugins/discover/public/plugin.tsx

```diff
@@ -427,7 +427,6 @@ export class DiscoverPlugin
           initialState,
         });
       },
-      embeddableType: SEARCH_EMBEDDABLE_TYPE,
```

Comment:
Hi @nreese, 

Does this change mean that `savedObjectType` will be used from now on instead of `embeddableType`? 

We still have quite a lot of references to `SEARCH_EMBEDDABLE_TYPE` in the code and I wonder if we should clean that up too.

Reply:
> Does this change mean that savedObjectType will be used from now on instead of embeddableType?

No. The embeddableType is still used when adding the panel. For example, discover still calls `addNewPanel` with `SEARCH_EMBEDDABLE_TYPE`. The only change is that `registerAddFromLibraryType` is registered against the saved object type instead of the embeddable type.